code/modules/research/techweb/all_nodes.dm

```diff
@@ -632,15 +632,6 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
-/datum/techweb_node/adv_beam_weapons
-	id = "adv_beam_weapons"
-	display_name = "Advanced Beam Weaponry"
-	description = "Various advanced beam weapons"
-	prereq_ids = list("beam_weapons")
-	design_ids = list("beamrifle")

```

Comment:
if you're going to basically remove the gun from the round atleast remove the design datum for it too

Reply:
Okay what do you mean exactly?

Reply:
this PR effectively results in beamrifle never being able to be made, which I'm assuming is the purpose of this PR. however, the design datum still exists in the code which is pointless because it cant be made so you should remove it.

Reply:
I'm not removing it entirely. I'm fine if it's one of a kind in some space ruin. I can't make maps so I just put that task up for other coders.

Reply:
remove the fucking design datum its the thing that makes it printable not the item itself

Reply:
where is it stored

Reply:
there should be a designs folder in techwebs or research